linux/ubuntu/versioned/releases

```diff
@@ -1,3 +1,4 @@
+2.164.6
```

Comment:
I wasn't sure if you wanted to keep them to build possibly more backwards compatible images (in case support for an older TFS version was removed?), so I left it as-is. Other than that, there's probably little value in building images for older agent versions. 🙂

Reply:
The old versions will still be available on docker hub.
The fix cfcouse will only aply to the new released version.

I am searching for a way tho have that version list automaticly updated but I did not found a way jet.

Reply:
Have you considered the [releases API](https://developer.github.com/v3/repos/releases/#list-releases-for-a-repository)? You can get the latest agent release `https://api.github.com/repos/microsoft/azure-pipelines-agent/releases/latest`.

Reply:
Last time I tried this there was a problem that the releases on github showed some wrong information:
https://github.com/microsoft/azure-pipelines-agent/issues/2688